pkg/apis/v1beta1/labels.go

```diff
@@ -46,12 +46,23 @@ const (
 	NodePoolHashAnnotationKey          = Group + "/nodepool-hash"
 )
 
+// Karpenter specific taints
+const (
+	DisruptingTaintKey = Group + "/disrupting"
```

Comment:
I'd suggest a single disruption taint with multiple values, rather than multiple taints. This enables users to take advantage of taint values to write more expressive tolerations, instead of writing explicit tolerations for each case. 

We have another chance to change this before v1, and I'd like to see how far we get with the more minimal approach.

Reply:
Synced offline, and changed to one taint with multiple values.